### ISSUE TITLE
Intense QC after joint calling of WGS sequenced by different platforms/options?

### ISSUE THREAD

**Issue by YiweiNiu** [shohei-kojima/MEGAnE#5]
Hi,

This page [Things to check before analysis](https://github.com/shohei-kojima/MEGAnE/wiki/Things-to-check-before-analysis#joint-calling-of-wgs-sequenced-by-different-platformsoptions=) says: "If you choose to make such joint callset, we strongly recommend intense QC after joint calling."

It would be helpful if you could describe this in detail.

In the section "Joint calling from 25x and 15x WGS datasets in BBJ" of the supplementary file of your [bioRxiv paper](https://www.biorxiv.org/content/10.1101/2022.03.25.485726v1.full), you jointly called the 15x WGS and 25x WGS, but I did not find how the QC was applied to improve the call quality.

Thank you in advance.

Bests,
Yiwei

**Comment by shohei-kojima** [comment]
Thank you very much for asking this, and sorry for the lack of explanation.
I have added the details in the Wiki page: https://github.com/shohei-kojima/MEGAnE/wiki/Things-to-check-before-analysis

In our manuscript, we made the joint callset of 15x and 25x, but did not do intense QC as you mentioned, because we used it to only see the number and positions of MEVs. When we use the genotype information, e.g. when we calculated LD, we used the joint callset from only 25x. When you merge two or more datasets with different sequencing settings, we recommend to use it only for checking the count of MEVs and positions. If you want to use the genotypes as well, we recommend you to only use datasets with the same sequencing settings, or make a joint callset from different sequencing settings and remove variants that are common but detected only in some specific sequencing settings, for example, by calculating HWE.

Thank you,
Koji
